src/LaboratoryResults-3-0/Test/_components/assert.organization.contact.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<nts:component xmlns="http://hl7.org/fhir" xmlns:nts="http://nictiz.nl/xsl/testscript">
+    <nts:parameter name="warningOnly" value="false"/>
+    <action>
+        <assert>
+            <description
+                value="Confirm the Bundle contains the right telecom (email/phone) information for {$resource}"/>
+            <direction value="{$direction}"/>
+            <expression value="Bundle.entry.resource.as({$resource}).telecom.where(system= '{$system}' and value  = '{$val}' and use = '{$use}').exists()"/>
```

Comment:
Just minor nitpicking: use 'value' instead of 'val', and remove superfluous whitespaces.